src/robot/libdocpkg/languages.py

```diff
@@ -0,0 +1,22 @@
+#  Copyright 2008-2015 Nokia Networks
+#  Copyright 2016-     Robot Framework Foundation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+# This is modified by invoke, do not edit by hand
+LANGUAGES = ['EN', 'FI', 'FR', 'IT', 'NL', 'PT-BR', 'PT-PT']
```

Comment:
It would be good to format this like below (incl. the trailing comma). The main reason is that there's a plan to format the code using ruff, and this format it wouldn't touch even if new languages were added. A smaller benefit is that this way it's easier to see what exact language has been added in a diff.
```python
LANGUAGES = [
    'EN',
    'FI',
    'FR',
    'IT',
    'NL',
    'PT-BR',
    'PT-PT',
]
```